base/test.jl

```diff
@@ -294,13 +368,15 @@ type TestSetException <: Exception
     pass::Int
     fail::Int
     error::Int
+    broken::Int
 end
 
 function Base.show(io::IO, ex::TestSetException)
     print(io, "Some tests did not pass: ")
     print(io, ex.pass,  " passed, ")
     print(io, ex.fail,  " failed, ")
-    print(io, ex.error, " errored.")
+    print(io, ex.error, " errored, ")
+    print(io, ex.broken, " broken.")
```

Comment:
True, the whole point of having `test_skip` as a separate thing was for reporting.